SPEC.md

```diff
@@ -118,32 +122,60 @@ An optional *path whitelist* can be provided, in which case all non-specified fi
-* Define the lifecycle of the container as all exit or first to exit
-* Define security requirements for a container. In particular is any isolation of users required between containers? What user does each application run under and can this be root (i.e. "real" root in the host).
+* Define the lifecycle of the pod as all exit or first to exit
+* Define security requirements for a pod. In particular is any isolation of users required between pods? What user does each application run under and can this be root (i.e. "real" root in the host).
 * Define how apps are supposed to communicate; can they/do they 'see' each other (a section in the apps perspective would help)?
 
 
 ## App Container Executor
 
-App Containers are a combination of a number of technologies which are not aware of each other.
-This specification attempts to define a reasonable subset of steps to accomplish a few goals:
-
-* Creating a filesystem hierarchy in which the app will execute
-* Running the app process inside of a combination of resource and namespace isolations
-* Executing the application inside of this environment
-
+The **App Container Executor** defines the process by which applications contained in ACIs are executed.
 There are two "perspectives" in this process.
-The "*executor*" perspective consists of the steps that the container executor must take to set up the containers. The "*app*" perspective is how the app processes inside the container see the environment.
+The "*executor*" perspective consists of the steps that the App Container Executor (ACE) must take to set up the environment for the pod and applications.
+The "*app*" perspective is how the app processes inside the pod see the environment.
 
 ### Executor Perspective
 
+#### Pod Manifest
+
+The deployable, executable unit in the App Container specification is the **pod**.
+A **pod** executes one or more apps inside a shared context.
+The shared context is defined as the conjunction of the following namespaces:
+- PID namespace
+- network namespace
+- mount namespace
+- IPC namespace
+- UTS namespace
+Each app in a pod will start chrooted into its own unique read-write rootfs before execution.
+The definition of the **pod** is a list of apps that should be launched together, along with isolators that should apply to the entire pod.
+This is codified in a [Pod Manifest](#pod-manifest-schema).
+Pod Manifest serve the role of _deployable templates_  and _runtime manifest_: a template can be a candidate for a series of transformations before execution.
+For example, a Pod Manifest might reference an app with a label requirement of `version=latest`, which another tool might subsequently resolve to a specific version.
+Another example would be that volumes are "late-bound" by the executor; alternatively, an executor might add annotations.
+The fully resolved, finalised pod manifest is exposed through the metadata service.
+
+TODO(jonboulle): define what a reified/finalised pod manifest is
```

Comment:
"Template" is certainly fine. "Reified" makes the most sense to me in the context of a template that is now concrete having its parameters defined.